app/src/main/kotlin/com/wire/android/mapper/SystemMessageContentMapper.kt

```diff
@@ -73,6 +73,7 @@ class SystemMessageContentMapper @Inject constructor(
         is MessageContent.ConversationProtocolChangedDuringACall -> mapConversationProtocolChangedDuringACall()
         is MessageContent.ConversationStartedUnverifiedWarning -> mapConversationCreatedUnverifiedWarning()
         is MessageContent.LegalHold -> mapLegalHoldMessage(content, message.senderUserId, members)
+        MessageContent.ConversationProtocolChangedDuringACall -> TODO()
```

Comment:
issue: something weird happened here probably when merging code and it got duplicated:
<img width="892" alt="image" src="https://github.com/wireapp/wire-android/assets/30429749/a6f9a6a4-c74d-49ec-9608-1076ddca8168">